LondonBritishLibrary/orient/BLorient482.xml

```diff
@@ -156,9 +231,22 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </list>
                      </additions>
                      
-                     
+                     <bindingDesc>
+                        <binding contemporary="false" xml:id="binding">
+                           <decoNote xml:id="b1">The manuscript has been resewn, and the original cover been replaced by 
+                              a wooden cover with blind-tooled leather quarter cover, very probably by
```

Comment:
This is unclear. Boards and cover are different elements in terms of binding structure. I would summaries it like this:
"The manuscript has been rebound in the Library. Both original wooden boards have been replaced, and a blind-tooled leather quarter cover added (with the gilded title and shelfmark on the spine). Sewing stations are used in an irregular way."  I personally used to record the information on shelfmarks and stamps under "extras"